src/Box/types.ts

```diff
@@ -22,6 +22,7 @@ export interface BoxProps extends BaseSpacingProps, React.HTMLAttributes<HTMLDiv
   elevation?: ElevationType;
   flexDirection?: FlexDirectionType;
   flexWrap?: FlexWrapType;
+  flexGap?: number;
```

Comment:
I'd suggest using the same name of the css property `gap`, as we're doing with the rest of props.

So these are equivalent:
```
<Box flexWrap="wrap" gap={10} />
<Box style={{ flexWrap: 'wrap', gap: 10 }} />
```